server/sonar-web/src/main/js/apps/overview/stickers/StickersModal.tsx

```diff
@@ -22,30 +22,39 @@ import Modal from '../../../components/controls/Modal';
 import StickerButton from './StickerButton';
 import StickerSnippet from './StickerSnippet';
 import StickerParams from './StickerParams';
-import { translate } from '../../../helpers/l10n';
 import { getStickerUrl, StickerType, StickerOptions } from './utils';
+import { Component } from '../../../app/types';
+import { translate } from '../../../helpers/l10n';
 import './styles.css';
 
+interface Props {
+  component: Component;
```

Comment:
You only need the component `key`, so type this as `component: { key: string }` or even better `component: string`. It'd simplify your tests (you won't need to type-cast props).